firefox_decrypt.py

```diff
@@ -768,6 +779,10 @@ def parse_sys_args():
 
     args = parser.parse_args()
 
+    if args.tabular:
+        args.format = 'csv'
+        args.delimiter = '\t'
```

Comment:
Yes, but for backwards compatibility we should keep `'` as the quotechar when outputting the `tabular` format. The old tabular used `\t` and `'` instead of `\t` and `"`.

The other comment was related to using `"` for strings instead of `'` to be consistent with the rest of the script.